test/testenv/verificationutils.go

```diff
@@ -676,3 +676,41 @@ func VerifyAppsDownloadedByInitContainer(deployment *Deployment, testenvInstance
 		}
 	}
 }
+
+// VerifyBundlePush verify that bundle push was pushed on all indexers
```

Comment:
It would be a good idea to have 2 separate methods for Deployer and Cluster Master so that the caller can verify individually.